src/chain/bls_emu.rs

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+//! Types emulating the BLS functionality until proper BLS lands
+use super::{delivery_group_size, NetworkEvent, ProofSet, SectionInfo};
+use crate::id::{FullId, PublicId};
+use parsec;
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    fmt,
+};
+
+#[derive(Clone, PartialEq, Eq)]
+pub struct PublicKeySet {
+    sec_info: SectionInfo,
+    threshold: usize,
+}
+
+#[derive(Clone, PartialEq, Eq)]
+pub struct PublicKey(PublicKeySet);
```

Comment:
Like with `Signature` - this is an emulated implementation of `threshold_crypto::PublicKey`. I want to keep the names the same as they are in `threshold_crypto`, so that when we merge BLS, we can just replace this module with `threshold_crypto` crate and keep the rest of the code mostly intact.

In this case, we would replace this in `lib.rs`:
`pub use crate::chain::bls_emu::PublicKey as BlsPublicKey;`
with this:
`pub use threshold_crypto::PublicKey as BlsPublicKey;`
when BLS is merged.